azure-pipelines.yml

```diff
@@ -83,10 +83,10 @@ stages:
         yarn bootstrap
         set CI=false
         yarn build
```

Comment:
Thanks to this commit that I've found something interesting.

We're testing `*.ts` then why do we build before the tests?

Reply:
Now we have some test cases executing .js files via child_process.fork:
https://github.com/nervosnetwork/neuron/blob/7d1b00b58fda76e282cbc92206b1550caa36eb29/packages/neuron-wallet/tests/utils/worker.test.ts#L39

Reply:
> Now we have some test cases executing .js files via child_process.fork:
> https://github.com/nervosnetwork/neuron/blob/7d1b00b58fda76e282cbc92206b1550caa36eb29/packages/neuron-wallet/tests/utils/worker.test.ts#L39

I was curious about adding the build step before. 🤔

Reply:
@yuche 
Could you please explain a bit more about why we need to change the build script to accommodate this test suit?

Reply:
Because it is a mock-up fixture, it is only for the test purpose. What if we directly save it as a `js` file for the fork function to refer to?

Reply:
> Because it is a mock-up fixture, it is only for the test purpose. What if we directly save it as a `js` file for the fork function to refer to?

We are using a pure javascript file for `child_process.fork`,  But that file needs to import the compiled file:

https://github.com/nervosnetwork/neuron/blob/7d1b00b58fda76e282cbc92206b1550caa36eb29/packages/neuron-wallet/tests/utils/fixtures/worker.js#L1-L6

To summarize, we need to test `child_process.fork`, but `child_process.fork` can only use .js files, and .js files can only import .js source code, so we have to compile the source code before testing.

Reply:
Right. I mean can we just save that specific worker fixture file as a pure javascript file instead of a typescript, so no need to involve the compilation?

Reply:
> > Now we have some test cases executing .js files via child_process.fork:
> 
> I was curious about adding the build step before. 🤔




> Right. I mean can we just save that specific worker fixture file as a pure javascript file instead of a typescript, so no need to involve the compilation?

I'm sorry I didn't get that, but the worker fixture file is now a pure javascript file.

Reply:
Ok, I assumed it was a ts file. 

How about using this technique. Does it work in our context? https://github.com/TypeStrong/ts-node/issues/619#issuecomment-511366883

Reply:
> Ok, I assumed it was a ts file.
> 
> How about using this technique. Does it work in our context? [TypeStrong/ts-node#619 (comment)](https://github.com/TypeStrong/ts-node/issues/619#issuecomment-511366883)

I haven't tested it yet, but this should work. But then we need to introduce an additional dependency: `ts-node`.

Perhaps I wasn't  describe it clear enough, but the current implementation actually runs on macOS and Linux, as well as Windows on the local machine, but the Windows instance of the Azure pipeline cleans up the files already created at each step, so the test fails.

Before this PR, the original procedure was to compile the source files before testing, but testing and compiling were in separate steps, except that this PR puts compiling and testing in one step in Windows CI.

Reply:
Makes sense to me

Reply:
If I got it right, this commit is to fix a test failure introduced by a PR that is not merged yet? Then why not merge this fix into that PR?

Reply:
And any documents indicate that `but the Windows instance of the Azure pipeline cleans up the files already created at each step, so the test fails.`? It is kinda weird since Mac and Linux don't do that.

Reply:
> And any documents indicate that `but the Windows instance of the Azure pipeline cleans up the files already created at each step, so the test fails.`? It is kinda weird since Mac and Linux don't do that.

No, but this PR does fix the problem, and I've tested it in my own repo. Please check out the two links in the PR description.

Reply:
> > And any documents indicate that `but the Windows instance of the Azure pipeline cleans up the files already created at each step, so the test fails.`? It is kinda weird since Mac and Linux don't do that.
> 
> No, but this PR does fix the problem, and I've tested it in my own repo. Please check out the two links in the PR description.

Don't mind that, I was just curious about that.

Reply:
But [this](https://github.com/nervosnetwork/neuron/pull/1811#discussion_r472677882) is my question, why not merge this into the PR introduced the test failure.

Reply:
> But [this](https://github.com/nervosnetwork/neuron/pull/1811#discussion_r472677882) is my question, why not merge this into the PR introduced the test failure.

I don't have access to the Neuron repository's branch protection rules, but I'm guessing we're supposed to be protecting the develop branch. since #1800 merge to develop branch, so we need to merge this PR into the develop branch for the CI to work.

Reply:
I mean merge this commit into https://github.com/nervosnetwork/neuron/tree/refactor/upgrade-to-electron-9

Reply:
It seems that this branch including extra commits to https://github.com/nervosnetwork/neuron/tree/refactor/upgrade-to-electron-9, I set the pr base back to develop